Tests/SentryTests/Transaction/SentrySpanTests.swift

```diff
@@ -140,8 +140,6 @@ class SentrySpanTests: XCTestCase {
     func testSerialization() {
         let span = fixture.getSut()
         
-        span.setExtra(value: fixture.extraValue, key: fixture.extraKey)
-        span.setTag(value: fixture.extraValue, key: fixture.extraKey)
```

Comment:
`h`: I think we should not remove this here. This test should still work.

Reply:
Added some extra logic to SentryTracer class. It didn't have its own tags and data before, but they were lost during serialization method (because SentryTracer was using its root span's ones)